sparse/coo/core.py

```diff
@@ -1436,6 +1436,22 @@ def __rmatmul__(self, other):
         except NotImplementedError:
             return NotImplemented
 
+    def __array_function__(self, func, types, args, kwargs):
+        import sparse as module
+        for submodule in func.__module__.split('.')[1:]:
+            try:
```

Comment:
The `try`/`catch` can be moved outside the `for`, with even the line `sparse_func = getattr(module, func.__name__)` inside it.

Reply:
I'm not sure I understand. What would be inside the `for`?